### ISSUE TITLE
When this nuget is added to my Blazor WASM project, the app does not load

### ISSUE THREAD

**Issue by pszeder** [StefH/Blazor.DownloadFileFast#3]
As the title says. I've got a working Blazor WASM project using .Net 6 and VS 2022. 
I download this nuget and then start the app and get the following exception on the console:

```
blazor.webassembly.js:1 System.AggregateException: One or more errors occurred. (Invalid JS call result type '3'.
Error: Invalid JS call result type '3'.
    at S (https://localhost:7171/_framework/blazor.webassembly.js:1:5038)
    at Object.invokeJSFromDotNet (https://localhost:7171/_framework/blazor.webassembly.js:1:3170)
    at Object.At [as invokeJSFromDotNet] (https://localhost:7171/_framework/blazor.webassembly.js:1:58414)
    at _mono_wasm_invoke_js_blazor (https://localhost:7171/_framework/dotnet.6.0.0.a1fp8dqnl9.js:1:193780)
    at wasm://wasm/00945abe:wasm-function[2588]:0x8065b
    at wasm://wasm/00945abe:wasm-function[2307]:0x689a7
    at wasm://wasm/00945abe:wasm-function[7015]:0x17bac4
    at wasm://wasm/00945abe:wasm-function[6830]:0x172bc5
    at wasm://wasm/00945abe:wasm-function[3524]:0xb47cc
    at wasm://wasm/00945abe:wasm-function[675]:0x16602)
 ---> Microsoft.JSInterop.JSException: Invalid JS call result type '3'.
Error: Invalid JS call result type '3'.
    at S (https://localhost:7171/_framework/blazor.webassembly.js:1:5038)
    at Object.invokeJSFromDotNet (https://localhost:7171/_framework/blazor.webassembly.js:1:3170)
    at Object.At [as invokeJSFromDotNet] (https://localhost:7171/_framework/blazor.webassembly.js:1:58414)
    at _mono_wasm_invoke_js_blazor (https://localhost:7171/_framework/dotnet.6.0.0.a1fp8dqnl9.js:1:193780)
    at wasm://wasm/00945abe:wasm-function[2588]:0x8065b
    at wasm://wasm/00945abe:wasm-function[2307]:0x689a7
    at wasm://wasm/00945abe:wasm-function[7015]:0x17bac4
    at wasm://wasm/00945abe:wasm-function[6830]:0x172bc5
    at wasm://wasm/00945abe:wasm-function[3524]:0xb47cc
    at wasm://wasm/00945abe:wasm-function[675]:0x16602
   at Microsoft.JSInterop.WebAssembly.WebAssemblyJSRuntime.InvokeJS(String identifier, String argsJson, JSCallResultType resultType, Int64 targetInstanceId)
   at Microsoft.JSInterop.JSInProcessRuntime.Invoke[IJSVoidResult](String identifier, Int64 targetInstanceId, Object[] args)
   at Microsoft.JSInterop.JSInProcessRuntime.Invoke[IJSVoidResult](String identifier, Object[] args)
   at Microsoft.JSInterop.JSInProcessRuntimeExtensions.InvokeVoid(IJSInProcessRuntime jsRuntime, String identifier, Object[] args)
   at Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer.AttachRootComponentToBrowser(Int32 componentId, String domElementSelector)
   at Microsoft.AspNetCore.Components.RenderTree.WebRenderer.AddRootComponent(Type componentType, String domElementSelector)
   at Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer.AddComponentAsync(Type componentType, ParameterView parameters, String domElementSelector)
   at Microsoft.AspNetCore.Components.WebAssembly.Hosting.WebAssemblyHost.<>c.<<RunAsyncCore>b__15_1>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Components.WebAssembly.Hosting.WebAssemblyHost.RunAsyncCore(CancellationToken cancellationToken, WebAssemblyCultureProvider cultureProvider)
   at Program.Main(String[] args) in Program.cs:line 41
   --- End of inner exception stack trace ---

```

`Program.cs:line 41` is ` builder.Build().RunAsync()`

I've tried it with an empty project as well, there it did not cause this issue. Unfortunately, I cannot share the project where the problem happened, but I'm happy to answer any questions.

**Comment by StefH** [comment]
Hello @pszeder<

In this PR (https://github.com/StefH/Blazor.DownloadFileFast/pull/4) I created a .NET example project, and it seems to work fine?

**Comment by pszeder** [comment]
Thank you for your reply @StefH !

I'm aware it works with the empty/template project:
>I've tried it with an empty project as well, there it did not cause this issue.

I compared other Nugets between the empty project and mine and started adding them to the empty project. 
After they had the same packages, I realised my project was still on 6.0 RC1 versions for MS Blazor nugets. Updating those fixed the problem, which is a bit strange, but means this issue can be closed.
